src/methods/redirect.ts

```diff
@@ -14,13 +14,15 @@ type RedirectParams<T, Params extends RouteParams> = Params extends EmptyObject
       clock?: Clock<T>;
       route: RouteInstance<Params>;
       query?: ((clock: T) => RouteQuery) | Store<RouteQuery> | RouteQuery;
+      replace?: boolean;
```

Comment:
I think better to change this type to  
```ts
replace?: ((clock: T) => boolean) | Store<boolean> | boolean;
```

I think that's more flexible functionality   

@sergeysova what do you think ?

Reply:
@js2me Makes sense